android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -366,14 +365,11 @@ private void startDownloadFromNetwork() {
     }
 
     private void downloadPageId() {
-        //Store the facebook page name in the shared preference from the database
-        if(SharedPreferencesUtil.getString(ConstantStrings.FACEBOOK_PAGE_NAME, null) == null) {
-            RealmList<SocialLink> socialLinks = event.getSocialLinks();
-            RealmResults<SocialLink> facebookPage = socialLinks.where().equalTo("name", "Facebook").findAll();
-            SocialLink facebookLink = facebookPage.get(0);
-            String pageName = facebookLink.getLink().substring(20);
-            SharedPreferencesUtil.putString(ConstantStrings.FACEBOOK_PAGE_NAME, pageName);
-        }
+
+        //Store the facebook and twitter page name in the shared preference from the database
+        storePageName("Facebook", ConstantStrings.FACEBOOK_PAGE_NAME);
```

Comment:
Instead of `Facebook` use `ConstantStrings.SOCIAL_LINK_FACEBOOK`.
Same for `Twitter`